src/rgw/rgw_rados.cc

```diff
@@ -1134,7 +1134,7 @@ int RGWPeriod::read_latest_epoch(RGWPeriodLatestEpochInfo& info,
   RGWObjectCtx obj_ctx(store);
   int ret = rgw_get_system_obj(store, obj_ctx, pool, oid, bl, objv, nullptr);
   if (ret < 0) {
-    ldout(cct, 1) << "error read_lastest_epoch " << pool << ":" << oid << dendl;
+    ldout(cct, 20) << "error read_lastest_epoch " << pool << ":" << oid << dendl;
```

Comment:
Is this all right? As an error, I think 20 is not appropriate. Why not 10?

Reply:
yeah i believe generic log pushed at log level =20 and error logs at log level=0.